src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java

```diff
@@ -0,0 +1,118 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.RegistrationEditSectionPage;
```

Comment:
Change the import statemement package to import from https://github.com/openmrs/openmrs-contrib-qaframework/tree/master/src/test/java/org/openmrs/contrib/qaframework/page to cater for the current changes